src/bdk_cli.rs

```diff
@@ -117,8 +119,32 @@ where
         timeout: wallet_opts.electrum_opts.timeout,
     });
 
-    // Fall back to Electrum config if Esplora config isn't provided
-    let config = config_esplora.unwrap_or(config_electrum);
+    #[cfg(feature = "compact_filters")]
+    let compact_filter_config: Option<AnyBlockchainConfig> = {
+        let peerconfig = BitcoinPeerConfig {
+            address: wallet_opts.compactfilter_opts.address.clone(),
+            socks5: None,
+            socks5_credentials: None,
+        };
+
+        Some(AnyBlockchainConfig::CompactFilters(
+            CompactFiltersBlockchainConfig {
+                peers: vec![peerconfig],
+                network,
+                storage_dir: wallet_opts.wallet.clone(),
```

Comment:
Ideally this should end up in the same data directory that we use for the database. Maybe you can refactor `prepare_home_dir()` a little bit to reuse the same path (currently it always appends `database.sled` at the end but obviously you don't want that for the compact filters dir)

Reply:
Not sure if I get it. Are you suggesting we should have different data directory for `electrum` `esplora` and `compact_filters`?

That would make sense. Currently it uses `database.sled` directory for all the cases in `.bdk-bitcoin` dir. We can have separate directories for each cases.

Reply:
datadir updated as `~/.bdk-bitcoin/compact_filters/<wallet-name>`